BrickController2/BrickController2.Android/PlatformServices/GameController/GameControllerService.cs

```diff
@@ -41,7 +41,12 @@ public bool OnKeyDown([GeneratedEnum] Keycode keyCode, KeyEvent e)
         {
             if ((((int)e.Source & (int)InputSourceType.Gamepad) == (int)InputSourceType.Gamepad) && e.RepeatCount == 0)
             {
-                GameControllerEventInternal?.Invoke(this, new GameControllerEventArgs(GameControllerEventType.Button, e.KeyCode.ToString(), 1.0F));
+                // https://developer.android.com/develop/ui/views/touch-and-input/game-controllers/multiple-controllers
+                // Note: On devices running Android 4.1(API level 16) and higher, you can obtain an input device’s descriptor using getDescriptor(), which returns a unique persistent
+                // string value for the input device.Unlike a device ID, the descriptor value won't change even if the input device is disconnected, reconnected, or reconfigured. 
+                string controllerDeviceId = $"{e.Device?.ControllerNumber ?? -1}";
```

Comment:
Nit pick: My suggestion is to move the logic to extension method so as it's not copy-pasted 3 times.

Reply:
I did it: 
https://github.com/vicocz/brickcontroller2/pull/75/commits/45781fe25326bd0c7a08bd176311f0e692a305b3